config/secrets.yml

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: aef132340a61d306ae9f90075b02c0934c138fccc9ca3e447417cc9f8b42cff77fc79bc0330c6dc33bc3c0f76a22af2115ff3178ec84c20db8e6c275abcac836
+  fb_APP_ID: 1577880519145437
+  fb_APP_SECRET: d7d7223d2ae0276720bea54d85f9eef5
+
+test:
+  secret_key_base: 1bf417a6d961c881d5eb247db0c031aa586e897f66009a99c554e84bc4949f044d1edede10c7fa509ce451412727de6100b7bc0d8f5c93e831827f5f7669af9b
+  fb_APP_ID: 1577880519145437
+  fb_APP_SECRET: d7d7223d2ae0276720bea54d85f9eef5
+
```

Comment:
Delete these facebook keys, reset them in the facebook application - you should not be pushing these to github.